crates/muvm/src/guest/mount.rs

```diff
@@ -88,8 +88,19 @@ fn mount_fex_rootfs() -> Result<()> {
         if Path::new(base).exists() {
             let json = format!("{{\"Config\":{{\"RootFS\":\"{dir_rootfs}\"}}}}\n");
             let path = base.to_string() + "/Config.json";
+            let host_dir = "/run/muvm-host".to_string() + base;
 
             make_tmpfs(base)?;
+            for entry in read_dir(host_dir).unwrap() {
+                let entry = entry.unwrap();
+                let file_name = entry.file_name().to_str().unwrap().to_owned();
+                if file_name == "Config.json" {
```

Comment:
```suggestion
                let file_name = entry.file_name();
                if file_name == "Config.json" {
```

https://doc.rust-lang.org/std/ffi/struct.OsString.html#impl-Deref-for-OsString
https://doc.rust-lang.org/std/path/struct.Path.html#impl-PartialEq%3C%26OsStr%3E-for-Path
https://doc.rust-lang.org/std/path/struct.Path.html#method.join
https://doc.rust-lang.org/std/path/struct.Path.html#impl-AsRef%3CPath%3E-for-OsString

Reply:
Thanks! (This was a leftover from before I was using `Path` and doing direct string concat instead)